src/InfoTrack.OAuth.Caching.DotNetStandard/InfoTrack.OAuth.Caching.DotNetStandard.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project file has some issues.

- There's now no NuGet package metadata. See the old .NET Core project file.
- There's now a reference to the .NET Framework Caching DLL. It should be using `Microsoft.Extensions.Caching.Memory`